common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -641,6 +641,7 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
   def masterPrometheusMetricPort: Int = get(MASTER_PROMETHEUS_PORT)
   def workerPrometheusMetricHost: String = get(WORKER_PROMETHEUS_HOST)
   def workerPrometheusMetricPort: Int = get(WORKER_PROMETHEUS_PORT)
+  def metricsExtraLables: Option[Seq[String]] = get(METRICS_EXTRA_LABELS)
```

Comment:
we can decode label at here
```suggestion
  def metricsExtraLables: Seq[(String, String)] = ...
```